File: Iniciante/2311/2311.c
#include <stdio.h>

int main()
{
    int N, i, j;
    char nome[20];
    double GD, pontuacao, max = 0.00, min = 10.00, soma = 0.00;
    scanf("%d", &N);
    for(i=1; i<=N; i++)
    {
        scanf("%s", nome);
        scanf("%lf", &GD);
        for(j=1; j<=7; j++)
        {
            scanf("%lf", &pontuacao);
            if(pontuacao > max) max = pontuacao;
            if(pontuacao < min) min = pontuacao;
            soma += pontuacao;
        }
        soma -= (max+min);
        soma *= GD;
        printf("%s %.2lf\n", nome, soma);
        max = 0.00, min = 10.00, soma = 0.00;
    }
    return 0;
}

File: Python/Iniciante/2311.py
# Problema 2311 - Beecrowd - Iniciante - Nível 2


# Recebe o número de pessoas
competidores = int(input())

# Inicia a lista vazia
nome = []

# Percorre o loop até chegar na quantidade total de competidores
for i in range(competidores):

    # Atribui o nome do competidor a lista vazia
    nome.append(input())

    # Recebe a dificuldade da tarefa
    dificuldade = float(input())

    # Mapeia a quantidade de notas transformando em uma lista
    notas = list(map(float, input().split()))

    # Anota o mínimo e o máximo da lista
    minimo = min(notas)
    maximo = max(notas)

    # Faz a conta e imprime o resultado com 2 casas
    nota_final = (sum(notas)-maximo-minimo)*dificuldade
    print(f'{nome[i]} {nota_final:.2f}')